modules/pytket-quantinuum/docs/intro.txt

```diff
@@ -17,6 +17,104 @@ and Windows. To install, run:
+
+* [1] If no value is specified then ``optimisation_level`` defaults to a value of 1.
+
+* [2] ``self.rebase_pass`` is a rebase that converts the circuit to the Quantinuum native gate set {Rz, PhasedX, ZZMax, ZZPhase}.
+
+* [3] ``auto_squash_pass`` has arguments ``auto_squash_pass({OpType.PhasedX,OpType.Rz})``
+
+* [4] ``SimplifyInitial`` has arguments  ``SimplifyInitial(allow_classical=False, create_all_qubits=True, xcirc=xcirc)``
+
+.. note::   If ``optimisation_level = 0`` the device constraints are solved but no additional optimisation is applied. Setting ``optimisation_level = 1`` applies some light optimisations to the circuit. More intensive optimisation is applied by level 2 at the expense of increased runtime.
+
+**Device Predicates**
+
+Circuits must satisfy the following predicates in order to run on the ``QuantinuumBackend``.
+
+* `NoSymbolsPredicate  <https://cqcl.github.io/tket/pytket/api/predicates.html#pytket.predicates.NoSymbolsPredicate>`_  - Parameterised gates must have numerical parameters when the circuit is executed.
+* `GateSetPredicate  <https://cqcl.github.io/tket/pytket/api/predicates.html#pytket.predicates.GateSetPredicate>`_ : To view supported Ops run ``QuantinuumBackend.backend_info.gate_set``
+* `MaxNQubitsPredicate(12) <https://cqcl.github.io/tket/pytket/api/predicates.html#pytket.predicates.MaxNQubitsPredicate>`_ for ``H1-1``, ``H1-2`` and ``H1-2E``. ``MaxNQubitsPredicate(20)`` for ``H1-1E``.
+
+
+**Additional Backend Capabilities**
+
+The Quantinuum ``Backend`` has a ``cost`` method. This calculates the cost (in H-series Quantum Credits) required to execute the circuit for the specified number of shots.  
+
+There are also ``login()`` and ``logout()`` methods. 
+After the login method is called the user will be prompted for their Quantinuum username and password.
+
+The ``process_circuits`` method for the QuantinuumBackend accepts the following additional keyword arguments.
+
+*  ``postprocess`` : boolean flag to allow classical postprocessing.
+*  ``noisy_simulation`` : boolean flag to specify whether the simulator should
+   perform noisy simulation with an error model (default value is ``True``).
+*  ``group`` : string identifier of a collection of jobs, can be used for usage tracking.
+*  ``max_batch_cost``: maximum HQC usable by submitted batch, default is 500.
+*  ``batch_id`` : first ``jobid`` of the batch to which this batch of circuits should be submitted. Job IDs can be retrieved from ResultHandle using ``backend.get_jobid(handle)``.
+*  ``close_batch`` : boolean flag to close the batch after the last circuit in the job, ``default=True``.
+
+For the Quantinuum ``Backend``, ``process_circuits`` returns a ResultHandle object containing a ``job_id`` and a postprocessing ( ``ppcirc``) circuit if there is one. The ``logout()`` method clears stored JSON web tokens and will have to sign in to access the Quantinuum API.
```

Comment:
Are there some words missing before "will have to sign in", e.g. "after calling this method the user" ...?